### PR TITLE
CM-303 - aac decoder

### PR DESCRIPTION
**Pull request by mrakes**
- initial branch/Hulafile

### CODE REVIEW

File: Hulafile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name 'fdk-aac'
+license 'MIT'
+
+dep 'base'
+build_dep 'autotools'
+
```
Comment (dressupgeekout) [inline Hulafile]: You ought to list `available_configurations %w[cosmos:lwm]` at a minimum

Comment (mrakes) [reply Hulafile]: yay! I knew you'd find something I missed! :)

File: Hulafile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+configure do
+  configure_flags = %W[
+    --prefix=#{prefix}
+    --host=#{host_platform.triple}
```
Comment (dressupgeekout) [inline Hulafile]: Also consider `--disable-static` to skip building and installing a static library.

File: Hulafile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  ]
+ 
+  sh %Q(NOCONFIGURE=yes ./autogen.sh)
```
Comment (dressupgeekout) [inline Hulafile]: The autogen script here does not respect NOCONFIGURE as an option, and it does not automatically perform ./configure;  therefore you can just `cmd './autogen.sh'`

File: Hulafile [hunk 4]
```diff
@@ -0,0 +1,23 @@
+name 'fdk-aac'
+license 'MIT'
```
Comment (dressupgeekout) [inline Hulafile]: I don't see the MIT license, only this weird Fraunhofer Society license

Comment (mrakes) [reply Hulafile]: ok. got that from @abahocodes and didn't check it. will update w/ everything else :)

File: Hulafile (hunk 2 above)
Comment (mrakes) [reply Hulafile]: done

File: Hulafile [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+available_configurations %w[cosmos:lwm]
+
+configure do
+  configure_flags = %W[
+    --prefix=#{prefix}
+    --host=#{host_platform.triple}
+    --disable-static
+  ]
+ 
+  sh %Q(./autogen.sh)
```
Comment (dressupgeekout) [inline Hulafile]: I meant `cmd './autogen.sh'` but whatever

**Pull request merged**